packages/neuron-ui/src/utils/scriptAndAddress.ts

```diff
@@ -1,15 +1,18 @@
+import { Script } from '@ckb-lumos/base'
```

Comment:
Annotate `Script` as `type Script` if it's introduced as type

Reply:
Added.